.github/dependabot.yml

```diff
@@ -8,6 +8,7 @@ updates:
     labels:
       - "domain: deps"
       - "no-changelog"
+      - "ci-condition: skip-regression"
```

Comment:
I forgot there are multiple "ecosystems" defined in this file. I think this label needs to be added for each of them.

Reply:
Thanks, added in https://github.com/vectordotdev/vector/pull/21507/commits/b9c7daee1676c76e65f8803ba2a1c143df188499